lumigator/python/mzai/backend/backend/jobs/submission.py

```diff
@@ -26,11 +17,21 @@ class RayJobEntrypoint(ABC):
 
     @property
     def command(self) -> str:
```

Comment:
```suggestion
    def build_ray_job_params
```

or similar since we now build up to the full command